src/include/main/client_context.h

```diff
@@ -24,8 +25,14 @@ class ClientContext {
      */
     KUZU_API ~ClientContext() = default;
 
+    /**
+     * @brief Returns whether the current query is interrupted or not.
+     */
+    KUZU_API bool isInterrupted() const { return interrupted; }
```

Comment:
Do we need to expose this as a public interface?

Reply:
Physical operator needs to access this field